WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { Dropdown, Header, Table } from 'semantic-ui-react';
+import I18n from '../../../lib/i18n';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import { apiV0Urls } from '../../../lib/requests/routes.js.erb';
+import Errored from '../../Requests/Errored';
+import Loading from '../../Requests/Loading';
+
+function GroupTable({ groupId }) {
+  const { data: roles, loading, error } = useLoadedData(apiV0Urls.userRoles.listOfGroup(
+    groupId,
+    { isActive: true, isGroupHidden: false, status: 'leader' },
+  ));
+
+  if (loading || loading) return <Loading />;
```

Comment:
:thinking:

Reply:
Oops 🤣 removed one now, thanks for noting.